dev-lang/mozart-stdlib/mozart-stdlib-2.0.1.ebuild

```diff
@@ -1,11 +1,11 @@
-# Copyright 1999-2020 Gentoo Authors
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=7
 
-inherit cmake-utils
-
 MY_PN=mozart2-stdlib
+BUILD_DIR="${S}/build" # TODO: really?
```

Comment:
This line can be dropped. The default BUILD_DIR used by the cmake eclass is fine.